wpimath/src/main/java/edu/wpi/first/math/kinematics/ChassisAccelerations.java

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.math.kinematics;
+
+import static edu.wpi.first.units.Units.MetersPerSecondPerSecond;
+import static edu.wpi.first.units.Units.RadiansPerSecondPerSecond;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.interpolation.Interpolatable;
+import edu.wpi.first.math.kinematics.proto.ChassisAccelerationsProto;
+import edu.wpi.first.math.kinematics.struct.ChassisAccelerationsStruct;
+import edu.wpi.first.units.measure.AngularAcceleration;
+import edu.wpi.first.units.measure.LinearAcceleration;
+import edu.wpi.first.util.protobuf.ProtobufSerializable;
+import edu.wpi.first.util.struct.StructSerializable;
+import java.util.Objects;
+
+/**
+ * Represents the acceleration of a robot chassis. Although this class contains similar members
+ * compared to a Twist2d, they do NOT represent the same thing. Whereas a Twist2d represents a
+ * change in pose w.r.t to the robot frame of reference, a ChassisAccelerations object represents a
+ * robot's velocity.
+ *
+ * <p>A strictly non-holonomic drivetrain, such as a differential drive, should never have an ay
+ * component because it can never move sideways. Holonomic drivetrains such as swerve and mecanum
+ * will often have all three components.
+ */
+public class ChassisAccelerations
+    implements ProtobufSerializable, StructSerializable, Interpolatable<ChassisAccelerations> {
+  /** Velocity along the x-axis in meters per second². (Fwd is +) */
+  public double ax;
```

Comment:
Why are these fields public and mutable?

Reply:
There's no invariants to enforce, so there's no reason to add setter/getter boilerplate. ChassisSpeeds's members are public and mutable for the same reason.